integrations/docker/build.sh

```diff
@@ -59,7 +59,12 @@ set -ex
 
 [[ -n $VERSION ]] || die "version cannot be empty"
 
-mb_space_before=$(df -m /var/lib/docker/ | awk 'FNR==2{print $3}')
+#Fix storage location of Docker images and containers for osx or linux
+if [[ $OSTYPE == 'darwin'* ]]; then
+    mb_space_before=$(df -m ~/Library/Containers/com.docker.docker/Data/vms/0/ | awk 'FNR==2{print $3}')
+else
+    mb_space_before=$(df -m /var/lib/docker/ | awk 'FNR==2{print $3}')
```

Comment:
Shouldn't we just set a variable up front based on `$OSTYPE` that points to the right location, and then we don't have to have these branches and the copy/pasted paths in multiple places....

Reply:
A very valid point. I will correct it. Thank you for your attention and review. :)